charts/argo-cd/templates/argocd-server/deployment.yaml

```diff
@@ -132,6 +136,13 @@ spec:
       {{- with .Values.server.extraContainers }}
         {{- toYaml . | nindent 6 }}
       {{- end }}
+      {{- if .Values.server.extensions }}
+      - name: argocd-extensions
+        image: ghcr.io/argoproj-labs/argocd-extensions:latest
```

Comment:
:thinking: if we want to parametrise this and pin it to a version. Not sure what upstream recommendation is TBH.

Reply:
makes sense, I've extended the extensions section to allow configuring the images and also optionally load the rollouts-extension